internal/api/v1/user/configs/configs.go

```diff
@@ -0,0 +1,38 @@
+package configs
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/bitcoin-sv/spv-wallet/models/response"
+	"github.com/go-resty/resty/v2"
+)
+
+const route = "api/v1/configs"
+
+type API struct {
+	addr string
+	cli  *resty.Client
+}
+
+func (a *API) SharedConfig(ctx context.Context) (*response.SharedConfig, error) {
+	var result response.SharedConfig
+	_, err := a.cli.
+		R().
+		SetContext(ctx).
+		SetResult(&result).
+		SetError(&models.SPVError{}).
```

Comment:
Sorry, I forgot to remove this - this `SetError` is unnecessary here because we set it up "globally" during resty initialisation.